composer/cli/launcher.py

```diff
@@ -80,6 +80,13 @@ def _get_parser():
         help=('If set, run the training script as a command (i.e. without `python`). '
               'Cannot be used in conjunction with `module_mode`.'),
     )
+    parser.add_argument('-k',
```

Comment:
could we get some more DANGER ZONE docstring? since this will kill _all_ python processes. and maybe log a warning about it if it is set.